src/ucp/api/ucp.h

```diff
@@ -333,6 +335,19 @@ typedef enum {
 
 /**
  * @ingroup UCP_DATATYPE
+ * @brief Generate a reusable identifier for Scatter-gather IOV data type.
```

Comment:
how would the lookup be performed on the reusable IOV then?
how is it different case than N sends with contig buffer?